core/src/components/card/test/basic/e2e.ts

```diff
@@ -1,10 +1,33 @@
-import { newE2EPage } from '@stencil/core/testing';
+import { E2EElement, newE2EPage } from '@stencil/core/testing';
 
 test('card: basic', async () => {
   const page = await newE2EPage({
     url: '/src/components/card/test/basic?ionic:_testing=true'
   });
 
+  await checkMode(await page.find('ion-card'));
+  await checkMode(await page.find('ion-card-content'));
+  await checkMode(await page.find('ion-card-header'));
+  await checkMode(await page.find('ion-card-subtitle'));
+  await checkMode(await page.find('ion-card-title'));
+
   const compare = await page.compareScreenshot();
   expect(compare).toMatchScreenshot();
 });
+
+// Given an element of type E2EElement, get the node name of the
+// element and the current mode to verify the mode class exists
+//
+// Example:
+// await checkMode(await page.find('ion-card-content'))
+// ios => expect(el).toHaveClass(`card-content-ios`);
+// mode => expect(el).toHaveClass(`card-content-md`);
+// -----------------------------------------------------------------
+async function checkMode(el: E2EElement, name?: string) {
```

Comment:
can we move this to the generic test utils file? also do you think we need to do this check anywhere else? it might be good to have tests that test the utility functions we write.